src/Symfony/Component/WebLink/Link.php

```diff
@@ -92,6 +92,8 @@ public function getAttributes(): array
 
     /**
      * {@inheritdoc}
+     *
+     * @return object
```

Comment:
`@return static`?

Reply:
Good catch, I actually missed propagating the original annotation, now fixed.
This means we now have `Node[]` instead of `array` also.